src/test/java/com/epam/tc/hw2/ex1/Exercise1Test.java

```diff
@@ -0,0 +1,93 @@
+package com.epam.tc.hw2.ex1;
+
+import com.epam.tc.hw2.AbstractChromeTest;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Test;
+
+public class Exercise1Test extends AbstractChromeTest {
+
+    public SoftAssertions softly = new SoftAssertions();
+
+    /** Assert Browser title. */
```

Comment:
Please, specify test group here, e.g., @Test(groups = {"<group_name>"}) and than use it in your suite.xml file.